README.md

```diff
@@ -4,6 +4,7 @@ Mathesar is a project to make databases easier to use for non-technical users. O
 
 We are currently in early development and hope to release an alpha version by late 2021. Please see the [Mathesar wiki](https://wiki.mathesar.org/) for more information about the project.
 
+Mathesar is open source under the GPLv3 license - see [LICENSE](LICENSE). It also contains derivatives of third-party open source modules. See the list and respective licenses in [THIRDPARTY](THIRDPARTY).
```

Comment:
```suggestion
Mathesar is open source under the GPLv3 license - see [LICENSE](LICENSE). It also contains derivatives of third-party open source modules licensed under the MIT license. See the list and respective licenses in [THIRDPARTY](THIRDPARTY).
```

Reply:
Please move this under a "License" header at the bottom of the file.

Reply:
Changes made in https://github.com/centerofci/mathesar/pull/363/commits/2e798b12b54db99d2857264ba3680d19ad44f23d